public/pages/Channels/components/details/ChannelDetails.tsx

```diff
@@ -126,76 +144,111 @@ export function ChannelDetails(props: ChannelDetailsProps) {
     },
   ];
 
+  const actionsAndMuteComponent = <EuiFlexGroup gutterSize="m" alignItems="flexEnd">
+    <EuiFlexItem />
+    <EuiFlexItem grow={false}>
+      {channel && (
+        <ChannelDetailsActions
+          channel={channel} />
+      )}
+    </EuiFlexItem>
+    <EuiFlexItem grow={false}>
+      {channel && (
+        <ModalConsumer>
+          {({ onShow }) => (
+            <EuiSmallButton
+              data-test-subj="channel-details-mute-button"
+              iconType={channel.is_enabled ? 'bellSlash' : 'bell'}
+              onClick={() => {
+                if (channel.is_enabled) {
+                  onShow(MuteChannelModal, {
+                    selected: [channel],
+                    setSelected: (selected: any[]) => setChannel(selected[0]),
+                  });
+                } else {
+                  const newChannel = { ...channel, is_enabled: true };
+                  servicesContext.notificationService
+                    .updateConfig(channel.config_id, newChannel)
+                    .then(() => {
+                      coreContext.notifications.toasts.addSuccess(
+                        `Channel ${channel.name} successfully unmuted.`
+                      );
+                      setChannel(newChannel);
+                    });
+                }
+              }}
+            >
+              {channel.is_enabled ? 'Mute channel' : 'Unmute channel'}
+            </EuiSmallButton>
+          )}
+        </ModalConsumer>
+      )}
+    </EuiFlexItem>
+  </EuiFlexGroup>;
+  const rightControls = [
+    {
+      renderComponent: (
+        actionsAndMuteComponent
+      ),
+    },
+    {
+      renderComponent: (
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <EuiSmallButton
+            data-test-subj="send-test-message-button"
+            onClick={sendTestMessage}
+            style={{ marginLeft: '10px' }}
+            disabled={!channel?.is_enabled}
+          >
+            Send test message
+          </EuiSmallButton>
+        </div>
+      ),
+    },
+  ];
+
+  const badgeComponent = <EuiFlexItem grow={false} style={{ paddingBottom: 5 }}>
+    {channel?.is_enabled === undefined ? null : channel.is_enabled ? (
+      <EuiHealth color="success">Active</EuiHealth>
+    ) : (
+      <EuiHealth color="subdued">Muted</EuiHealth>
+    )}
+  </EuiFlexItem>;
+
+  const badgeControls = [
+    {
+      renderComponent: (
+        badgeComponent
+      ),
+    },
+  ];
+
   return (
     <>
-      <EuiGlobalToastList
-        toasts={toasts}
-        dismissToast={(removedToast) => {
-          setToasts(toasts.filter((toast) => toast.id !== removedToast.id));
-        }}
-        toastLifeTimeMs={60000}
-      />
-
-      <EuiFlexGroup
-        alignItems="center"
-        gutterSize="m"
-        style={{ maxWidth: 1316 }}
+      <PageHeader
+        appRightControls={getUseUpdatedUx() ? rightControls : []}
```

Comment:
Don't need this check here since the page header component takes care of it. Check similarly in all other places too

Reply:
addressed